lighthouse-core/index.js

```diff
@@ -51,7 +51,7 @@ module.exports = function(url, flags, configJSON) {
 
     flags = flags || {};
 
-    // set logging preferences, assume quiet
+    // set logging preferences, assume quiet/silent
```

Comment:
in this case, the fallback is `error` which === `quiet` so we don't need to mention silent.